src/views/projectExplorer/libraryList.ts

```diff
@@ -47,23 +50,36 @@ export default class LibraryList extends ProjectExplorerTreeItem {
 
           case `USR`:
             let libraryType = LibraryType.defaultUserLibrary;
+            let position: Position | undefined;
+
             if (state?.preUsrlibl && state.preUsrlibl.includes(library.libraryInfo.name)) {
+              const listLength = state.preUsrlibl.length;
```

Comment:
`state.preUsrlibl` can still have variables here. This means you need to filter our the ones that start with '&':
```typescript
const listLength = state.preUsrlibl.filter(lib => !lib.startsWith('&')).length;
```

![image](https://github.com/IBM/vscode-ibmi-projectexplorer/assets/32170854/70a331bd-8433-4985-b2e5-f600e52dbcf1)